### PR TITLE
Update cloud-sql sample apps to deploy to Cloud Run using Jib.

### LINKED ISSUE #4337: Instructions to deploy to Cloud Run don't work in Cloud SQL sample
## In which file did you encounter the issue?

[cloud-sql/mysql/servlet/README.md](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/cloud-sql/mysql/servlet/README.md)

### Did you change the file? If so, how?

No changes.

## Describe the issue
I followed the steps described in the README to deploy the sample to Cloud Run. However, when the app was deployed, the vote counts weren't displaying and popup messages were displaying raw HTML. The network requests return 200, and there didn't seem to be any errors when checking logs for Cloud SQL or Cloud Run.

Screen Shot 2020-12-02 at 1 49 55 PM

image

### PR DESCRIPTION
**Pull request by kurtisvg**
Fixes #4337

Updates instructions for Cloud SQL samples to use Jib when deploying to Cloud Run. 

- [X] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)
- [X] `pom.xml` parent set to latest `shared-configuration`
- [X] Appropriate changes to README are included in PR
- [X] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: cloud-sql/mysql/servlet/README.md [hunk 1]
```diff
@@ -70,10 +70,10 @@ mvn clean package appengine:deploy
 See the [Cloud Run documentation](https://cloud.google.com/run/docs/configuring/connect-cloudsql)
 for more details on connecting a Cloud Run service to Cloud SQL.
 
-1. Build the container image:
+1. Build the container image using [Jib](https://cloud.google.com/java/getting-started/jib):
 
   ```sh
-  gcloud builds submit --tag gcr.io/[YOUR_PROJECT_ID]/run-mysql
+    mvn compile com.google.cloud.tools:jib-maven-pluginbuild \ -Dimage=gcr.io/[YOUR_PROJECT_ID]/run-mysql
```
Comment (averikitsch) [inline cloud-sql/mysql/servlet/README.md:76]:
```suggestion
    mvn compile com.google.cloud.tools:jib-maven-plugin:2.7.0:build \ 
      -Dimage=gcr.io/[YOUR_PROJECT_ID]/run-mysql
```

File: cloud-sql/mysql/servlet/README.md [hunk 2]
```diff
@@ -70,10 +70,10 @@ mvn clean package appengine:deploy
...
   ```
 
 2. Deploy the service to Cloud Run:
```
Comment (averikitsch) [inline cloud-sql/mysql/servlet/README.md:79]: Can you add flags:
```
    --platform managed \
    --allow-unauthenticated \
   --region [REGION]
```

For step 3, we prefer using `--update-env-vars` and if you want more clarity you can use the flag multiple times like:
```
--update-env-vars CLOUD_SQL_CONNECTION_NAME=[INSTANCE_CONNECTION_NAME] \
--update-env-vars DB_USER=[MY_DB_USER],DB_PASS=[MY_DB_PASS],DB_NAME=[MY_DB]
```

**Review by averikitsch — CHANGES_REQUESTED** [review]

File: cloud-sql/mysql/servlet/README.md (hunk 2 above)
Comment (kurtisvg) [reply cloud-sql/mysql/servlet/README.md:80]: Done + combined steps 2+3.

**Review by averikitsch — APPROVED** [review]
Thanks for updating!

**Review by lesv — APPROVED** [review]

**Pull request merged**